src/generation.rs

```diff
@@ -2102,6 +2081,40 @@ impl<'a> WasmWrapper<'a> {
         if let Some(from_native) = self.from_native {
             gen_scope.wasm(types, self.ident).push_impl(from_native);
         }
+        if let Some(as_ref) = self.as_ref {
+            gen_scope.wasm(types, self.ident).push_impl(as_ref);
+        }
+    }
+
+    /// native_name is &str since we need to possibly prepend namespacing
+    /// and where we're calling it we'd have to construct a RustType where we
+    /// didn't have to before, but we already had the string.
+    fn add_conversion_methods(&mut self, native_name: &str) {
+        let mut from_wasm = codegen::Impl::new(self.ident.to_string());
+        from_wasm
+            .impl_trait(format!("From<{}>", native_name))
+            .new_fn("from")
+            .arg("native", native_name)
+            .ret("Self")
+            .line("Self(native)");
+            //.line(format!("Self({}(native))", native_name));
```

Comment:
Do we need this comment?

Reply:
I can remove it. It was simply the code from the wasm wrapper creation code moved directly.

Reply:
Removed.